### ISSUE TITLE
函数调用问题

### ISSUE THREAD

**Issue by lhz0635** [locuslab/TCN#59]
作者您好!我正尝试使用TCN的前向传播获取序列的输出，但是我在传参的过程中，出现错误。我在tensorflow中用placeholder开辟input传给TCN，在session中实例化，但是出现Tensor no len ；如果给TCN直接传递相同size的实参，就可以获取输出。请问，这是TCN的特性吗？必须传递实参吗？
部分代码如下：
input = tf.placeholder("float", [None, 20, 10])
model=TCN(model = TCN(input_channels, n_classes, channel_sizes, kernel_size=kernel_size, dropout=args.dropout))
**logits = model(Input)**    这一句就报错，我意识到可能是input第一维有none的原因，input是在session中确定最终的size为[64,20,10]。
恳请您给出高见，期待您的回复，祝您生活愉快。
